.chloggen/multi-instrumentation-command-flag.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
I would prefer to mark it as a braking change

Reply:
It should definitely be a breaking change.